server/command.go

```diff
@@ -83,6 +83,6 @@ func (p *Plugin) listTopicsToChannel() *model.CommandResponse {
 	}
 	return &model.CommandResponse{
 		ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL,
-		Text:         fmt.Sprintf("%s", resp),
+		Text:         (resp),
```

Comment:
Can this just be
```suggestion
		Text:         resp,
```
?

Reply:
It work both way, but your suggestion makes code more understandable. Made those changes.